carousel.js

```diff
@@ -0,0 +1,83 @@
+const carouselImagewrapper = document.getElementById("carousel__image-wrapper");
+const carouselWrapper = document.getElementById("carousel-wrapper");
+const carouselTitle = document.getElementById("carousel__title-wrapper");
+
+const carouselImg = [
+    {
+        id: "carousel__image",
+        class: "carousel__image",
+        srs: "images/carousel_img_1.png",
+        alt: "eggs and chicken",
+    },
+    {
+        id: "carousel__image",
+        class: "carousel__image",
+        srs: "images/carousel_img_2.png",
+        alt: "something more",
+    },
+    {
+        id: "carousel__image",
+        class: "carousel__image",
+        srs: "images/carousel_img_3.png",
+        alt: "something more",
+    }
+];
+
+for (let i = 0; i < carouselImg.length; i++) {
+    const img_creater = document.createElement("img")
```

Comment:
Naming should be in the [camelcase notation](https://techterms.com/definition/camelcase)
Also what does this variable store:
Functionality for creating `img`?
HTML element `img`?